python/ray/dag/tests/test_accelerated_dag.py

```diff
@@ -90,6 +98,60 @@ def test_regular_args(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_multi_args(ray_start_regular):
+    # Test passing multiple args to .execute.
+    a = Actor.remote(0)
+    with InputNode() as i:
+        dag = a.inc_two.bind(i[0], i[1])
```

Comment:
Same here, would be good to make sure that i[0] and i[1] are passed in the right order.

Reply:
updated